pearcej/cpp4python
# Remove extra identifier for each image (cpp4python-v2)

# Description

As in the issue described, there were extra identifiers for each image across the book I removed the extra identifiers and leave the default ones.
## Related Issue
fix #161 




## How Has This Been Tested?
1. Make changes locally.
2. Verify the changes.
![image](https://github.com/pearcej/cpp4python/assets/117699634/5344d429-340c-4495-a5cf-9bb4684baaae)
![image](https://github.com/pearcej/cpp4python/assets/117699634/1273d8a8-2317-4862-87d4-bd068b8950ed)
